src/smart-components/Channel/components/ChannelHeader/channel-header.scss

```diff
@@ -34,7 +34,7 @@
     overflow: hidden;
     text-overflow: ellipsis;
     white-space: nowrap;
-    max-width: 780px;
+    max-width: calc(100vw - 600px);
```

Comment:
I think to make it more screen friendly you can use %age or rem instead of pixel.

Reply:
thanks, will do that